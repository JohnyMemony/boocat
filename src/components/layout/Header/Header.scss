@import "../../../assets/styles/variables";

.bc-header {
  display: flex;
  position: fixed;
  z-index: 100;
  left: 0;
  top: 0;
  width: 100%;
  padding: 0 $contentPadding;
  height: $headerHeight;
  align-items: center;
  box-shadow: 0 2px 2px 0 rgba(0, 0, 0, .14), 0 3px 1px -2px rgba(0, 0, 0, .2), 0 1px 5px 0 rgba(0, 0, 0, .12);
  background-color: $white;

  &__logo {
    width: 80px;
    height: 80px;
    background: url(../../../assets/images/svg/logo.svg) 50% 50% no-repeat;
    background-size: 100% 100%;
  }
}