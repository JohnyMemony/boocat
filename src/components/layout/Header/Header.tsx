import * as React from 'react';
import {Link} from 'react-router-dom';
import {ROUTES} from '../../../configs/routes';

import './Header.scss';

export default function Header() {
  return (
    <header className="header">
      <Link to={ROUTES.ROOT} className="header__logo"/>
    </header>
  );
}
