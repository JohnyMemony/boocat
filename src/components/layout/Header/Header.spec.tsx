import * as React from 'react';
import {shallow} from 'enzyme';
import {BrowserRouter as Router} from 'react-router-dom';
import * as renderer from 'react-test-renderer';
import Header from './Header';

describe('Header', () => {
  describe('Header initial', () => {
    const HeaderComponent = renderer
      .create(
        <Router>
          <Header/>
        </Router>
      )
      .toJSON();

    it('RendersProperly', () => {
      expect(HeaderComponent).toMatchSnapshot();
    });
  });
});
