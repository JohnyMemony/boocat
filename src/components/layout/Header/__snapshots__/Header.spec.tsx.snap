// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Header initial RendersProperly 1`] = `
<header
  className="bc-header"
>
  <a
    className="bc-header__logo"
    href="/"
    onClick={[Function]}
  />
</header>
`;
